### PR TITLE
remove mapper config

### PR DESCRIPTION
**Pull request by ngchwanlii**
Bean creation initiated using @Mapper annotation in mapper.

### CODE REVIEW

**Comment by cliangdev** [comment]
@ngchwanlii It works without using spring bean?

**Comment by ngchwanlii** [comment]
1. remove mapper config, @Mapper(componentModel="spring") =  can initiate spring bean
2. Basic React + Spring integration test

File: account/src/main/java/com/caldevsupplychain/account/config/CorsConfig.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
+
+@Configuration
+public class CorsConfig {
+	@Bean
+	public WebMvcConfigurer corsConfigurer() {
+		return new WebMvcConfigurerAdapter() {
+			@Override
+			public void addCorsMappings(CorsRegistry registry) {
+				registry.addMapping("/api/account/v1/**").allowedOrigins("http://localhost:3000");
```
Comment (calvin-liang) [inline account/src/main/java/com/caldevsupplychain/account/config/CorsConfig.java:16]: It's better to move a url to a property file;
```
ublic class CorsConfig {
       @Value("${cors.url}")
       private String corsUrl;

 	@Bean
 	public WebMvcConfigurer corsConfigurer() {
 		return new WebMvcConfigurerAdapter() {
 			@Override
 			public void addCorsMappings(CorsRegistry registry) {
 				registry.addMapping("/api/account/v1/**").allowedOrigins(corsUrl);
        }
```
and `cors.url` should be defined in application.yaml

File: account/src/main/java/com/caldevsupplychain/account/config/ShiroConfig.java [hunk 2]
```diff
@@ -75,4 +76,6 @@ public DefaultAdvisorAutoProxyCreator defaultAdvisorAutoProxyCreator() {
 		daapc.setProxyTargetClass(true);
 		return daapc;
 	}
+
```
Comment (calvin-liang) [inline account/src/main/java/com/caldevsupplychain/account/config/ShiroConfig.java:79]: remove the extract lines

File: account/src/main/java/com/caldevsupplychain/account/config/CorsConfig.java (hunk 1 above)
Comment (ngchwanlii) [reply account/src/main/java/com/caldevsupplychain/account/config/CorsConfig.java:16]: fixed

File: account/src/main/java/com/caldevsupplychain/account/config/ShiroConfig.java (hunk 2 above)
Comment (ngchwanlii) [reply account/src/main/java/com/caldevsupplychain/account/config/ShiroConfig.java:79]: fixed

**Pull request closed without merging**